### PR TITLE
Feature/426687 lapcap problem with upload

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

**Review by johnbenjamin-defra — APPROVED** [review]

File: src/EPR.Calculator.Frontend/Helpers/CsvFileHelper.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static ErrorViewModel ValidateCSV(IFormFile fileUpload)
                 return errorViewModel = new ErrorViewModel() { DOMElementId = string.Empty, ErrorMessage = StaticHelpers.FileMustBeCSV };
             }
 
-            if (fileUpload.Length > StaticHelpers.MaxFileSize)
+            if (fileUpload.Length > StaticHelpers.MaxFileSize && !isLapCapData)
```
Comment (udaydefra) [inline src/EPR.Calculator.Frontend/Helpers/CsvFileHelper.cs:26]: Not sure why you need this `isLapCapData`

File: src/EPR.Calculator.Frontend/Helpers/CsvFileHelper.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace EPR.Calculator.Frontend.Helpers
 {
     public static class CsvFileHelper
     {
-        public static ErrorViewModel ValidateCSV(IFormFile fileUpload)
+        public static ErrorViewModel ValidateCSV(IFormFile fileUpload, bool isLapCapData = false)
```
Comment (udaydefra) [inline src/EPR.Calculator.Frontend/Helpers/CsvFileHelper.cs:13]: Not sure why you need this `isLapCapData`

**Review by udaydefra — DISMISSED** [review]
LGTM

**Review by johnbenjamin-defra — DISMISSED** [review]

**Review by rekhadhankhar2018 — APPROVED** [review]

File: src/EPR.Calculator.Frontend/Views/Shared/_ValidationErrorsListPartial.cshtml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿@model List<CreateLapcapDataErrorDto>;
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.Frontend/Views/Shared/_ValidationErrorsListPartial.cshtml:1]: I think this partial view is shared with default parameters. I think we do not need a partial view in this case as the model is different. Could you please check?

**Review by johnbenjamin-defra — CHANGES_REQUESTED** [review]

**Review by rekhadhankhar2018 — DISMISSED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by rekhadhankhar2018 — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by rekhadhankhar2018 — APPROVED** [review]

**Review by Mazar-Shaik — APPROVED** [review]

**Pull request closed without merging**